### DISCUSSION TITLE
Secure connection error when installing python via pyenv

### DISCUSSION THREAD

**Discussion by TMCG123** [community/community#133004] [Programming Help]
### Body

My understanding is that Windows 7 is not a reason for not being able to install new version of Python via pyenv.
When I run `pyenv install 3.10.0` (other versions also fail) is get the following in the console:
![utltimate_django](https://github.com/user-attachments/assets/9babe1e7-36eb-431b-a367-a933d505803e)

Can anyone suggest how I can approach this problem?

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by davevad93** [comment]
Hi @TMCG123 ,

Windows 7 only supports up to `Python` 3.8.6, from version 3.9 onwards you need at least Windows 8 as stated on the [Python website](https://www.python.org/downloads/windows/). 

![0](https://github.com/user-attachments/assets/65e537f5-6c81-458f-9147-23e50d5839d1)

I have an old Windows 7 laptop that I use offline with python 3.8 and have never been able to install a newer version there.

**Reply by davevad93** [reply]
Although yours looks like a connection issue that shouldn't be related to `pyenv`.
